nexus/src/app/mod.rs

```diff
@@ -200,6 +202,10 @@ impl Nexus {
         &self.tunables
     }
 
+    pub fn rack_id(&self) -> Uuid {
```

Comment:
It looks like this is only used for the integration tests?  If so, what do you think of putting it into TestInterfaces?  I'm a _little_ worried that by having this be so first-class and easily accessed, people will conflate the rack with the control plane itself.

Reply:
Done